examples/parallel/distributed_backend_simple.py

```diff
@@ -49,7 +49,7 @@ def long_running_function(i):
 ###############################################################################
 # The verbose messages below show that the backend is indeed the
 # dask.distributed one
-with joblib.parallel_backend("dask"):
+with joblib.parallel_config("dask"):
```

Comment:
Since parallel_config has many params, I think it's better to use keywords args when illustrating its usage.
```suggestion
with joblib.parallel_config(backend="dask"):
```